test/native-tests/test_native.sh

```diff
@@ -42,13 +47,7 @@ TEST () {
     fi
 }
 
-RES=$(c++ --version)
-if [[ ! $RES =~ "Apple LLVM" ]]; then
-    FIND_CLANG
-else
-    CC="cc"
-    CXX="c++"
-fi
+FIND_CLANG
```

Comment:
well, if we remove this, we won't be able to test gcc/g++ compiling a test app. You don't need to include PAL.h to use that. define the extern inside the C/xx file and linker will find it for you.